ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/uniqueness/model/UniquenessCheckResponse.kt

```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.ledger.utxo.uniqueness.model
+
+import net.corda.v5.application.crypto.DigitalSignatureAndMetadata
+
+/**
+ * This is a response class that wraps the result of the uniqueness checking and a potential signature,
+ * if the request was processed successfully.
+ *
+ * @property result The result produced by the uniqueness checker
+ * @property signature Contains the signature provided by the uniqueness checker. Might be `null`
```

Comment:
_Might be_, or _will be_?

Reply:
Yeah the phrasing here is not the best. I meant "This field might be null, in case the result is a failure", I'll rephrase it.